Makefile

```diff
@@ -15,11 +15,24 @@ check:
 check-dummy:
 	BUILD_DUMMY_WASM_BINARY= cargo check
 
+# Pseudo private target is invoked by public targets for different chains
+--try-runtime:
+	RUST_LOG=runtime=trace,try-runtime::cli=trace,executor=trace \
+		cargo run \
+		--bin=zeitgeist \
+		--features=parachain,try-runtime \
+		try-runtime \
+		--execution=Native \
```

Comment:
Yeah with `Native` we have no `wasm::stripped` cases anymore, but more helpful error messages instead. 👍 I noticed it while benchmarking stuff. That's why I used this (`./target/release/zeitgeist benchmark pallet --chain dev --execution=native --wasm-execution=compiled --heap-pages=4096 --pallet "*" --extrinsic "*" --steps 1 --repeat 0 --detailed-log-output`) command to quick check the benchmarks. https://github.com/zeitgeistpm/zeitgeist/issues/822